src/app_config.cpp

```diff
@@ -349,7 +368,117 @@ bool config_deserialize(const char *json)
 
 bool config_deserialize(DynamicJsonDocument &doc)
 {
-  return user_config.deserialize(doc);
+  bool config_modified = user_config.deserialize(doc);
+
+  #if ENABLE_CONFIG_CHANGE_NOTIFICATION
+  // Update EVSE config
+  // Update the EVSE setting flags, a little low level, may move later
+  if(doc.containsKey("diode_check"))
+  {
+    bool enable = doc["diode_check"];
+    if(enable != evse.isDiodeCheckEnabled()) {
+      evse.enableDiodeCheck(enable);
+      config_modified = true;
+      DBUGLN("diode_check changed");
+    }
+  }
+
+  if(doc.containsKey("gfci_check"))
+  {
+    bool enable = doc["gfci_check"];
+    if(enable != evse.isGfiTestEnabled()) {
+      evse.enableGfiTestCheck(enable);
+      config_modified = true;
+      DBUGLN("gfci_check changed");
+    }
+  }
+
+  if(doc.containsKey("ground_check"))
+  {
+    bool enable = doc["ground_check"];
+    if(enable != evse.isGroundCheckEnabled()) {
+      evse.enableGroundCheck(enable);
+      config_modified = true;
+      DBUGLN("ground_check changed");
+    }
+  }
+
+  if(doc.containsKey("relay_check"))
+  {
+    bool enable = doc["relay_check"];
+    if(enable != evse.isStuckRelayCheckEnabled()) {
+      evse.enableStuckRelayCheck(enable);
+      config_modified = true;
+      DBUGLN("relay_check changed");
+    }
+  }
+
+  if(doc.containsKey("vent_check"))
+  {
+    bool enable = doc["vent_check"];
+    if(enable != evse.isVentRequiredEnabled()) {
+      evse.enableVentRequired(enable);
+      config_modified = true;
+      DBUGLN("vent_check changed");
+    }
+  }
+
+  if(doc.containsKey("temp_check"))
+  {
+    bool enable = doc["temp_check"];
+    if(enable != evse.isTemperatureCheckEnabled()) {
+      evse.enableTemperatureCheck(enable);
+      config_modified = true;
+      DBUGLN("temp_check changed");
+    }
+  }
+
+  if(doc.containsKey("service"))
+  {
+    EvseMonitor::ServiceLevel service = static_cast<EvseMonitor::ServiceLevel>(doc["service"].as<uint8_t>());
+    if(service != evse.getServiceLevel()) {
+      evse.setServiceLevel(service);
+      config_modified = true;
+      DBUGLN("service changed");
+    }
+  }
+
+  if(doc.containsKey("max_current_soft"))
+  {
+    long current = doc["max_current_soft"];
+    if(current != evse.getMaxConfiguredCurrent()) {
+      evse.setMaxConfiguredCurrent(current);
+      config_modified = true;
+      DBUGLN("max_current_soft changed");
+    }
+  }
+
+  if(doc.containsKey("scale") && doc.containsKey("offset"))
```

Comment:
@mathieucarbou related to other comments about RAPI being removed, the current calibration can be set using the scale and offset